Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -78,6 +78,55 @@ export async function DeployUsingMSDeploy(webDeployPkg, webAppName, publishingPr
     }
 }
 
+function argStringToArray(argString): string[] {
```

Comment:
After this changes, Will our task support Additional Arguments with double spaces ?